pytestqt/qtbot.py

```diff
@@ -285,14 +287,14 @@ def waitSignal(self, signal=None, timeout=1000, raising=None):
                 raising = True
             else:
                 raising = _parse_ini_boolean(raising_val)
-        blocker = SignalBlocker(timeout=timeout, raising=raising)
+        blocker = SignalBlocker(timeout=timeout, raising=raising, callback=callback)
         if signal is not None:
             blocker.connect(signal)
         return blocker
 
     wait_signal = waitSignal  # pep-8 alias
 
-    def waitSignals(self, signals=None, timeout=1000, raising=None):
+    def waitSignals(self, signals=None, timeout=1000, force_order = "none", callbacks = None, raising=None):
```

Comment:
Also I think I'd rename `force_order` to `order`, and the parameters should be at the end as well.